dts/arm/nxp/nxp_mcxc_common.dtsi

```diff
@@ -253,6 +253,52 @@
 			status = "disabled";
 			#pwm-cells = <3>;
 		};
+
+		lptmr0: lptmr@40040000 {
+			compatible = "nxp,lptmr";
+			reg = <0x40040000 0x1000>;
+			interrupts = <28 0>;
+			clock-frequency = <1000>;
+			prescaler = <1>;
+			prescale-glitch-filter = <1>;
+			clk-source = <1>;
+			resolution = <16>;
+			status = "disabled";
+		};
+
+		rtc: rtc@4003d000 {
+			compatible = "nxp,kinetis-rtc";
+			reg = <0x4003d000 0x1000>;
+			interrupts = <20 0>, <21 0>;
+			interrupt-names = "alarm", "seconds";
+			clock-frequency = <32768>;
+			prescaler = <32768>;
+			status = "disabled";
+		};
+
+		pit0: pit@40037000 {
+			compatible = "nxp,pit";
+			reg = <0x40037000 0x1000>;
+			clocks = <&sim KINETIS_SIM_BUS_CLK 0x103c 23>;
+			max-load-value = <0xffffffff>;
+			status = "disabled";
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			pit0_channel0: pit0_channel@0 {
+				compatible = "nxp,pit-channel";
+				reg = <0>;
+				interrupts = <22 0>;
+				status = "disabled";
+			};
+
+			pit0_channel1: pit0_channel@1 {
+				compatible = "nxp,pit-channel";
+				reg = <1>;
+				interrupts = <22 0>;
```

Comment:
you can put the interrupt on the parent pit node, see RT series DT for example, no need for config_shared_interrupts

Reply:
Thank you. I have placed the interrupts to the parent node.